.github/workflows/main.yml

```diff
@@ -18,7 +21,8 @@ jobs:
           ref: v0.4.2
       - name: Sleep for 30s
         run: sleep 30
-      - name: Algolia crawler creation and reindex
+      - name: Algolia crawler creation and reindex on preview (Pull Request)
```

Comment:
you could probably deduplicate this code by having two step that output a different site-url (seems to work)

https://www.edwardthomson.com/blog/github_actions_15_sharing_data_between_steps.html

```
    steps:
      - id: main
        run: export FOO="https://algolia-ga-actions-netlify.netlify.app/"
      - id: pr
        run: export FOO="https://deploy-preview-${{ github.event.pull_request.number }}--algolia-ga-actions-netlify.netlify.app/"
```